supplier_stock/models/supplier_stock.py

```diff
@@ -77,13 +77,19 @@ class SupplierStock(models.Model):
     price_unit_base = fields.Float(
         string='Unit Price (Base)',
         digits=dp.get_precision('Account'),
-        compute='_compute_price_base'
+        compute='_compute_price_base',
+        store=True,
     )
     image_small = fields.Binary(
         'Image',
         related='product_id.product_tmpl_id.image_small',
         readonly=True,
     )
+    image_medium = fields.Binary(
```

Comment:
@eHanseFrank , I believe you have added this field to oa_partner_stock_offer as well, maybe we do not need to add it here?